assets/php/examples/master_detail/project_list.php

```diff
@@ -10,9 +10,12 @@ class ProjectListForm extends QForm {
 		protected $dtgProjects;
 					
 		protected function Form_Create() {
-			// Instantiate the Meta DataGrid
 			//$this->objDefaultWaitIcon = new QWaitIcon($this);
-			$this->dtgProjects = new ProjectDataGrid($this);
+
+			// Instantiate the DataGrid
+			// It is a simple QDataGrid because the generated class ProjectList
+			// is inherited from the QDataGrid2 now
+			$this->dtgProjects = new QDataGrid($this);
```

Comment:
I've tried to move to the QDataGrid2 here, but it does not support all QDataGrid features used in this example. It is quite complicated with datagrid in datagrid in dtagrid )